src/repository/entity/OwnerEntity.java

```diff
@@ -0,0 +1,75 @@
+package repository.entity;
+
+import java.time.LocalDate;
+import java.util.UUID;
+
+public class OwnerEntity extends BaseEntity {
+
+    public OwnerEntity() {
+    }
+
+    public OwnerEntity(UUID ownerId, String name, LocalDate birthDate) {
+        super(ownerId, name, birthDate);
+    }
+
+    @Override
+    public UUID getId() {
+        return super.getId();
+    }
+
+    @Override
+    public void setId(UUID id) {
+        super.setId(id);
+    }
+
+    @Override
+    public String getName() {
+        return super.getName();
+    }
+
+    @Override
+    public void setName(String name) {
+        super.setName(name);
+    }
+
+    @Override
+    public LocalDate getBirthDate() {
+        return super.getBirthDate();
+    }
+
+    @Override
+    public void setBirthDate(LocalDate birthDate) {
+        super.setBirthDate(birthDate);
+    }
+
+    @Override
+    public String toString() {
+        return "ID: "
```

Comment:
It is a good practice to print name of class, so we know how which class we are looking at